impeller/typographer/glyph_atlas.cc

```diff
@@ -23,9 +23,14 @@ void GlyphAtlas::SetTexture(std::shared_ptr<Texture> texture) {
 }
 
 void GlyphAtlas::AddTypefaceGlyphPosition(FontGlyphPair pair, Rect rect) {
+  has_color_ = pair.font.HasColor();
```

Comment:
I think this needs to be `|=`.

I'm also thinking we might want this to be an enum, but I guess we'll see. I'm looking at adding an enum for SDF.

Reply:
Oops